README.md

```diff
@@ -223,6 +223,16 @@ In response to the request, the Oak Manager sends back to the caller details
 about the gRPC endpoint of the newly created Oak Node, initialized with the Oak
 Module and policy configuration specified in the request.
 
+The following sequence diagram shows a basic flow of requests between a client,
+the Oak Manager and an Oak Node.
+
+<img src="docs/BasicFlow.png" width="850">
```

Comment:
Could you add the source to build the first diagram somewhere, and / or a link to the google doc for the second diagram? Or some other way to allow others (at least Googlers) to edit them easily.

Reply:
Added HTML comments with provenance URLs, and dumped the current sequence diagram text into `docs/BasicFlow.seq`